src/libraries/System.Private.CoreLib/src/System/SearchValues/SearchValues.cs

```diff
@@ -48,8 +48,8 @@ public static SearchValues<byte> Create(ReadOnlySpan<byte> values)
                 Debug.Assert(values.Length is 2 or 3 or 4 or 5);
                 return values.Length switch
                 {
-                    2 => new Any2ByteSearchValues(values),
-                    3 => new Any3ByteSearchValues(values),
+                    2 => new Any2SearchValues<byte, byte>(values),
```

Comment:
I assume we could do a similar masking approach for bytes just as we do for chars and are just choosing not to at this point since it's not worth the extra code?

Reply:
Yes, we could use the same approach for bytes, and also for "regular chars" for ARM (I only added it to the `Packed` paths that are X86-specific). I just chose not to do it yet. For bytes, I wasn't sure we had enough cases where we care (given no regex).

I'd imagine the relative perf difference should be similar there if we choose to do it.